tests/knownKeys

```diff
@@ -129,3 +129,11 @@ config_defaultPictureInPictureScreenEdgeInsets
 config_pictureInPictureDefaultSizePercent
 config_pictureInPictureDefaultAspectRatio
 config_pictureInPictureSnapMode
+config_enable_emergency_call_while_sim_locked
+config_device_respects_hold_carrier_config
+config_use_voip_mode_for_ims
+config_carrier_settings_enable
+config_carrier_volte_provisioning_required
+dtmf_type_enabled
```

Comment:
doesn't exist in aosp 11?

Reply:
please refer to this [link](https://android.googlesource.com/platform/packages/services/Telephony/+/refs/heads/master/res/values/config.xml#26).

Reply:
Unless I'm mistaken, knownKeys only check framework-res overlays, not overlays of other APKs.

Reply:
well, they're ok if someone targets com.android.phone for instance. In any case, the first three values exist in framework base.

Reply:
well, yes